src/main/scala/Main.scala

```diff
@@ -2,7 +2,9 @@ import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.{Sink, Source}
 import akka.util.ByteString
-import io.dronekit.cloud.ExternalCommandFlow
+import com.typesafe.scalalogging.Logger
```

Comment:
Dunno, it's kind of handy to have a complete example of how to use it. But yes, maybe it should be moved to an IT folder.